resources/idlharness.js

```diff
@@ -2676,6 +2676,7 @@ IdlInterface.prototype.test_members = function()
         }
 
         if (!exposed_in(exposure_set(member, this.exposureSet))) {
+            var desc = member.type === "operation" ? member.toString() : member.name;
```

Comment:
Looking at the results, there's both "ML interface: operation createContextSync(optional MLContextOptions)" and "ML interface: operation createContextSync(GPUDevice)" tests. Just like the tests you've deduplicated below, these tests are also just the same with different names.

Since the arguments don't matter, I think we should just use `member.name`, not `member.toString()`.